src/main/java/com/segment/analytics/android/integrations/adobeanalytics/AdobeIntegration.java

```diff
@@ -136,6 +137,12 @@ public String key() {
   }
 
   static class PlaybackDelegate implements MediaHeartbeatDelegate {
+    long initialTime;
+    long playheadPosition;
```

Comment:
In the future it seems like it might be unclear for a newcomer to understand what these fields are. We should document what they represent.